README.md

```diff
@@ -101,6 +101,13 @@ var rdfGraph = ds.Graph;
 new VDS.RDF.Writing.CompressingTurtleWriter().Save(rdfGraph, myIoWriter);
 ```
 
+### Supported Versions
+| Bindings Version | CASE Version | .NET Version    |
+|------------------|--------------|-----------------|
+| 0.1.0            | 1.0.0        | Framework 4.7.2 |
+| 0.2.0            | 1.0.0        | Standard 2.0    |
+| 0.3.0 (latest)   | 1.1.0        | Standard 2.0    |
+
 ## Known limitations
 
 **Properties with no declared target type** are omitted from the bindings. As of version 1.0.0, this only affects the
```

Comment:
Is this table still accurate as of 1.1.0?

Reply:
Good catch, table has been updated in [b56fdf6](https://github.com/ciphertechsolutions/CASE-Bindings-CSharp/pull/5/commits/b56fdf6d2c66b565f58d0962eee986d5f88e8974)